services/apps/src/apps/routes/domain/permission.clj

```diff
@@ -0,0 +1,18 @@
+(ns apps.routes.domain.permission
+  (:use [common-swagger-api.schema :only [describe]]
+        [schema.core :only [defschema optional-key enum]])
+  (:import [java.util UUID]))
+
+(defschema AppIdList
+  {:apps (describe [String] "A List of app identifiers")})
+
+(defschema UserPermission
+  {:user       (describe String "The user ID")
+   :permission (describe String "The permission level assigned to the user")})
```

Comment:
I considered making it an `enum`, but Grouper supports arbitrary action names. That said, we are sticking to `read`, `write`, `own`, and the empty string right now. I suppose we could make it an enum and change it later if necessary.